contract_manager/store/contracts/EvmPriceFeedContracts.yaml

```diff
@@ -169,9 +166,6 @@
 - chain: coredao_testnet
   address: "0x8D254a21b3C86D32F7179855531CE99164721933"
   type: EvmPriceFeedContract
-- chain: tomochain_testnet
```

Comment:
this network is renamed to victious but their testnet is a reset. so we need to redeploy later.